inc/saiacl.h

```diff
@@ -227,21 +254,149 @@ typedef enum _sai_acl_table_group_attr_t
     SAI_ACL_TABLE_GROUP_ATTR_START,
 
     /**
-     * @brief Dummy attribute
+     * @brief ACL stage
      *
-     * @type sai_int32_t
-     * @flags READ_ONLY
+     * @type sai_acl_stage_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_ACL_TABLE_GROUP_ATTR_ACL_STAGE = SAI_ACL_TABLE_GROUP_ATTR_START,
+
+    /**
+     * @brief List of ACL bind points where this group will be applied
+     *
+     * @type sai_s32_list_t sai_acl_bind_point_t
+     * @flags CREATE_ONLY
+     * @default empty
      */
-    SAI_ACL_TABLE_GROUP_ATTR_DUMMY = SAI_ACL_TABLE_GROUP_ATTR_START,
+    SAI_ACL_TABLE_GROUP_ATTR_ACL_BIND_POINT_LIST,
```

Comment:
I do not know if we need this or not since ACL_TABLE has the bind points. Assume you want to bind an ACL table group to a bind point, then all the acl tables should include that bind point as their types.